cmd/agent/main.go

```diff
@@ -209,6 +209,7 @@ func main() {
 			sig := <-signalChannel
 
 			if sig == os.Interrupt || sig == syscall.SIGTERM {
+				log.Printf("execution canceled by '%v' signal!", sig)
```

Comment:
We already know this because we have the `log.Printf("Captured %v...", sig)` below.